.github/workflows/api-integration-tests.yml

```diff
@@ -53,7 +55,7 @@ jobs:
           coverage: none
 
       - name: Setup BATS
-        uses: mig4/setup-bats@v1.2.0
+        run: npm install -g bats
```

Comment:
😱 but bats is a bash program. I think `sudo apt-get install -y bats` would be better